### PR TITLE
Move lint task into gulp

### PR DESCRIPTION
**Pull request by FredKSchott**
/cc @justinfagnani

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -33,6 +35,20 @@ gulp.task('clean', (done) => {
 
 gulp.task('build-all', ['clean', 'init', 'build']);
 
+gulp.task('lint', function (cb) {
+  exec('depcheck . --ignores "generator-polymer-init"', function (err, stdout, stderr) {
```
Comment (justinfagnani) [inline gulpfile.js]: add: TODO: import depcheck directly so it doesn't need to be a global install

Comment (justinfagnani) [inline gulpfile.js]: fyi: https://github.com/mcasimir/gulp-depcheck

**Comment by justinfagnani** [comment]
LGTM

File: gulpfile.js (hunk 1 above)
Comment (FredKSchott) [inline gulpfile.js]: re global install: whoops, I completely missed that implication. I think this is a step backwards then (npm was including the dev dependency in the path when run). i'll close this until it's ready to go again.

**Pull request closed without merging**